common/libs/globalFuncs.js

```diff
@@ -0,0 +1,14 @@
+export function makeBlob(mime, str) {
+  str = typeof str === 'object' ? JSON.stringify(str) : str;
+  if (str === null) return '';
+  const blob = new Blob([str], {
+    type: mime
+  });
+  return window.URL.createObjectURL(blob);
+}
+
+export const scryptSettings = {
+  n: 1024
```

Comment:
Might be 2048 eventually per @tayvano's request, but leaving as the v3 settings for now.